src/plugins/lua/bayes_expiry.lua

```diff
@@ -72,7 +72,7 @@ local function check_redis_classifier(cls, cfg)
       end
     end
 
-    if not symbol_spam or not symbol_ham or not type(expiry) == 'number' then
```

Comment:
What's different here?

Reply:
`not` has precedence over `==`, so `not type(expiry)` is always boolean `false`

Reply:
Oh, blame me then!